dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -1687,6 +1765,46 @@ protected synchronized void writeStreamOutput(SystemLogHolder holder, String txt
         holder.originalStream.println(txt);
     }
 
+    /**
+     * This method is accessed via reflection by the com.ibm.ws.logging.internal.osgi.stackjoiner.MethodProxy class
+     * It will be called on any Throwable.printStackTrace(PrintStream) invocation.
```

Comment:
I'd suggest the following comment...

This method is accessed by bytecode injected into the start of the Throwable.printStackTrace(PrintStream) method.  When this method returns true, printStackTrace will skip over the normal printStackTrace method body.  When this method returns false, printStackTrace will run the normal printStackTrace method body.